sklearn/neural_network/tests/test_mlp.py

```diff
@@ -591,6 +593,29 @@ def test_warm_start():
         assert_raise_message(ValueError, message, clf.fit, X, y_i)
 
 
+@ignore_warnings(category=FutureWarning)
+def test_warm_start_full():
+    # Test to make sure it is fitting for more than 1 iteration
+    X = Xboston
+    y = yboston
+
+    mlp = MLPRegressor(solver='adam', max_iter=150, hidden_layer_sizes=50,
+                       random_state=1, warm_start='full')
+    with ignore_warnings(category=ConvergenceWarning):
+        mlp.fit(X, y)
+
+    assert mlp.score(X, y) > -4.
```

Comment:
makes more sense to me to test the number of iterations...

Reply:
Yeah I forgot about the existence of `n_iter_`